src/components/SQFormDialog/SQFormDialog.js

```diff
@@ -47,43 +34,21 @@ function SQFormDialog({
       validationSchema={validationYupSchema}
       validateOnMount={true}
     >
-      <Dialog
+      <SQFormDialogInner
+        cancelButtonText={cancelButtonText}
+        children={children}
         disableBackdropClick={disableBackdropClick}
+        isDisabled={isDisabled}
+        isOpen={isOpen}
         maxWidth={maxWidth}
-        open={isOpen}
-        TransitionComponent={Transition}
         onClose={onClose}
-      >
-        <Form>
-          <DialogTitle disableTypography={true}>
-            <Typography variant="h4">{title}</Typography>
-          </DialogTitle>
-          <DialogContent dividers={true}>
-            <Grid
-              {...muiGridProps}
-              container
-              spacing={muiGridProps.spacing || 2}
-            >
-              {children}
-            </Grid>
-          </DialogContent>
-          <DialogActions className="sqFormDialog__actions">
-            <RoundedButton
-              title={cancelButtonText}
-              onClick={onClose}
-              color="secondary"
-              variant="outlined"
-            >
-              {cancelButtonText}
-            </RoundedButton>
-            {onSave && (
-              <SQFormButton title={saveButtonText}>
-                {saveButtonText}
-              </SQFormButton>
-            )}
-          </DialogActions>
-        </Form>
-      </Dialog>
+        onSave={onSave}
+        saveButtonText={saveButtonText}
+        title={title}
+        enableReinitialize={false}
```

Comment:
Can we expose `enableReinitialize` as a prop to the consumer? 
This prop has been needed more and more in SQForm (It's exposed there)

Reply:
oh I think this is my mistake... good catch

Reply:
yeah I messed up a couple props getting passed down to the inner dialog component that only Formik needs.

Reply:
done!